src/visualization/visualization.jl

```diff
@@ -8,4 +8,12 @@ include("types.jl")
 include("utilities.jl")
 include("recipes_plots.jl")
 
+# Until Julia v1.9 is the minimum required version for Trixi.jl, we still support Requires.jl
+@static if VERSION >= v"1.10.0-DEV.1288" || VERSION >= v"1.9.1"
```

Comment:
```suggestion
# Until Julia v1.9 is the minimum required version for Trixi.jl, we still support Requires.jl
# We do not check `isdefined(Base, :get_extension)` since Julia v1.9.0
# does not load package extensions when their dependency is loaded from
# the main environment
@static if VERSION >= v"1.9.1"
```